src/lib.rs

```diff
@@ -174,7 +185,9 @@ pub trait Actor: 'static + Send + Sized {
 
     /// Called as soon as the actor has been started.
     #[allow(unused_variables)]
-    async fn started(&mut self, ctx: &mut Context<Self>) {}
+    async fn started(&mut self, mailbox: &mut Mailbox<Self>) -> Result<(), Self::Stop> {
```

Comment:
This might introduce a strange pattern of 
```rust
async fn started(..) -> Result<(), Result<(), Error>>
```

Reply:
I've never had `Result` as the `Stop` value of an actor. Seems weird to me?

Reply:
If we just make it return `Stop`, we'd have to trait bound it to somehow construct a default value so it is an optional function.

Reply:
What I would mean is that if you wanted to exit early, you might want to return an error. However, you might want to always return some Ok value from stopped, e.g some statistics. Therefore, to satisfy this api, stop would be a result.

Reply:
Hmm, should we use `ControlFlow` here instead of `Result` to not have the "error" notion in it?

Reply:
Another way to look at this is that a double result is not actually bad. See https://sled.rs/errors.html.

Reply:
Fair enough! I'm OK with keeping it like this, then.